recirq/time_crystals/dtcsimulation_test.py

```diff
@@ -0,0 +1,104 @@
+# Copyright 2022 Google
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import recirq.time_crystals as time_crystals
+import numpy as np
+import itertools
+from typing import List
+import cirq
+
+
+def test_run_comparison_experiment():
```

Comment:
It's unusual that this test does not assert anything.

Reply:
I agree that it's odd, but frankly I'm unsure what to assert. The current tests serve to run every possible branch of the code, but checking that the code generated the correct data structures would require... generating those data structures again in a different way to test against. I have added some cursory asserts, but they don't claim much.